src/app.py

```diff
@@ -284,6 +285,48 @@ def get_depth_at_point(task_id: str) -> Response:
     }), OK)
 
 
+@app.route('/scenarios/medusa/<int:scenario_id>', methods=["GET"])
+@check_celery_alive
+def get_rainfall_information(scenario_id: int) -> Response:
+    """
+    Find a list of parameters below at a particular point for a given completed model output task:
+        - antecedent_dry_days
+        - average_rain_intensity
+        - event_duration
+        - rainfall_ph
+
+    Supported methods: GET
+    Required query param values: "scenario_id": str
+
+    Parameters
+    ----------
+    scenario_id: int
+        The scenario ID of the pollution model run
+
+    Returns
+    -------
+    Response
+        Returns JSON response in the form below:
+        {
+            "antecedent_dry_days": int,
+            "average_rain_intensity": int,
+            "event_duration": int,
+            "rainfall_ph": int
```

Comment:
This also includes geometry, which makes me think that maybe it should actually be served in geoserver. Get in contact with me to talk about this at some point this week and I can demonstrate what I mean